lib/providers/auth_provider.dart

```diff
@@ -64,9 +64,11 @@ class AuthProvider extends ChangeNotifier {
   logOut() {
     _helper.disconnect();
     _status = AuthStatus.SIGNED_OUT;
+    notifyListeners();
   }
 
   Future<String> logIn() async {
+    // TODO: handle unexpected behaviour (e.g. clicking cancel)
     try {
```

Comment:
There is a TODO here as well

Reply:
This is not related to this issue (and I think more of an issue for the oath_client plugin), so I'll leave it in